packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.15.0`
+
+- Added the secure credentials support for Extenders API [#1306](https://github.com/zowe/vscode-extension-for-zowe/pull/1306). Thanks @JillieBeanSim
+- Improved the Command Palette by adding 'Zowe Explorer:` before all commands that are related to the extension. Removed some commands from the palette that caused issues [#1308](https://github.com/zowe/vscode-extension-for-zowe/pull/1308). Thanks @lauren-li
+- Updated Theia Tests. Now you need to have Zowe CLI 6.30.0 and the latest .vsix file in the `theia/plugins` folder to run Theia tests [#1268](https://github.com/zowe/vscode-extension-for-zowe/pull/1268). Thanks @deepali-hub
+- Fixed the issue that preventied the `issue STOP command` function from executing correctly [#1304](https://github.com/zowe/vscode-extension-for-zowe/pull/1304). Thanks
+  @nickImbirev
+- Fixed the issue that caused the Add Profile icon to disapper [#1307](https://github.com/zowe/vscode-extension-for-zowe/pull/1307). Thanks @lauren-li
+- Fixed the vulnerability in NPM Audit [#1309](https://github.com/zowe/vscode-extension-for-zowe/pull/1309). Thanks @JillieBeanSim
+- Fixed the issue that doubled the occurence of the port prompt [#1298](https://github.com/zowe/vscode-extension-for-zowe/pull/1298). Thanks @katelynienaber
+- Fixed the issue that affected VS Code views after deleting a key binding in Zowe Explorer [#1310](https://github.com/zowe/vscode-extension-for-zowe/pull/1310). @crawr
+
```

Comment:
I am missing the changelog for this PR: https://github.com/zowe/vscode-extension-for-zowe/pull/1282

Reply:
Looks like the PR added its own changelog bullet point so that should be added/merged to this changelog PR (said the same in Slack)